qa/logger-usage/build.gradle

```diff
@@ -0,0 +1,43 @@
+import org.elasticsearch.gradle.precommit.PrecommitTasks
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+group = 'org.elasticsearch.qa'
+apply plugin: 'elasticsearch.build'
+
+dependencies {
+  compile 'org.ow2.asm:asm-debug-all:5.0.4'
+  testCompile "org.elasticsearch.test:framework:${version}"
+}
+
+// the main files are actually test files, so use the appropriate forbidden api sigs
+forbiddenApisMain {
+  bundledSignatures = ['jdk-unsafe', 'jdk-deprecated']
+  signaturesURLs = [PrecommitTasks.getResource('/forbidden/all-signatures.txt'),
+                    PrecommitTasks.getResource('/forbidden/test-signatures.txt')]
+}
+
+// TODO: should we have licenses for our test deps?
+dependencyLicenses.enabled = false
```

Comment:
If it were in buildSrc I don't think it'd have to do this. OTOH it should probably follow the same rules and declare the dependency.